content/docs/cve/cveIn-PersonExamProcess

```diff
@@ -0,0 +1,122 @@
+---
```

Comment:
I think you need to add a reference to this page in https://github.com/HamStudy/examtools-docs/blob/master/content/docs/cve/_index.md and maybe also https://github.com/HamStudy/examtools-docs/blob/master/content/docs/_index.md